stats/big-simulation.js

```diff
@@ -0,0 +1,37 @@
+let Simulation = require('./simulation')
+let modules = require('./strategies')
+let feeRate = 56 * 100
+let results = []
+
+let list = require('./moneypot-hotwallet.json')
+
+// for each strategy
+for (var name in modules) {
+  let f = modules[name]
+  let simulation = new Simulation(name, f, feeRate)
+
+  console.log(name)
+  let i = 0
+  for (var value of list) {
+    i++
+
+    if (i % 1000 === 0) {
+      console.log(i)
+    }
+    let txo = {
+      address: 'A',
+      value: Math.abs(value)
```

Comment:
please don't merge this yet. There is a bug; because this has no script, simulation.js thinks this is a change output (if it's being sent), so all utxos are always returned back to the "wallet".